word_guess.rb

```diff
@@ -0,0 +1,212 @@
+require 'faker'
+require 'colorize'
+
+# Used 2 classes
+# A 'Word Guess Game' has a 'word'
+
+class WordGuessGame
+  attr_accessor :word, :tries, :lives, :user_guess
+
+  # Word Guess Game has:
+  # nouns: word, levels, lives
+  # & verbs: active play
+
+  def initialize(word)
+    @word = word
+    @tries = tries
+    @lives = ""
+    @user_guess
+  end
+
+  def easy_level
+    @lives = 5
+    @user_guess = ""
+    puts active_play(@user_guess)
```

Comment:
in this line (and in the last line of the method `hard_level`, you two wrote `puts active_play(@user_guess)`. This implies that you want to print the returned value of the method `active_play`, which in most cases is `nil` or at least isn't important. You should probably just invoke the method without puts, as `active_play(@user_guess)`